test/exercism/user_test.rb

```diff
@@ -18,8 +18,13 @@ def test_flipper_id_is_username
     assert_equal 'spocktocat', user.flipper_id
   end
 
+  def github_user(id, username, email, avatar_url)
+    GitHubUser.new("id" => id, "login" => username, "email" => email, "avatar_url" => avatar_url)
+  end
```

Comment:
Rather than: 
```ruby
 GitHubUser.new("id" => id, "login" => username, "email" => email, "avatar_url" => avatar_url)
```

You could use different ruby hash syntax and write it as:
```ruby
 GitHubUser.new(id: id, login: username, email: email, avatar_url: avatar_url)
```
Which has less "noise".
(Although technically not identical if you really want string keys, but I don't think you do.)

Then once you have this shorter, nicer way to define `GitHubUsers` you can do away with the now even more redundant indirection.

`github_user(id, username, email, avatar_url)`
becomes
`GitHubUser.new(id: id, login: username, email: email, avatar_url: avatar_url)`

So maybe the 2 things were related. :smile: